docs/0___preamble.adoc

```diff
@@ -93,68 +93,65 @@ This document defines the Functional Mock-up Interface (FMI),
 version 3.0 to (a) exchange dynamic models between tools and (b) define tool
 coupling for dynamic system simulation environments.
 
-_FMI for Model Exchange (<<fmi-for-model-exchange>>)_
-
-The intention is that a modeling environment can generate a C code representation of
-a dynamic system model that can be utilized by other modeling and simulation environments.
-Models are described by differential,
-algebraic and discrete equations with time-, state- and step-events.
-If the C code describes a continuous system,
-this system is solved with the integrators of the environment where it is used.
-The models to be treated by this interface can be large for usage in offline or online simulation,
-or they can be used in embedded control systems on microprocessors.
-
 _FMI for Co-Simulation (<<fmi-for-co-simulation>>)_
 
 The intention is to provide an interface standard for coupling
-of simulation tools in a co-simulation environment.
+of simulation models or tools in a co-simulation environment.
 The data exchange between subsystems is restricted to discrete communication points.
 In the time between two communication points,
-the subsystems are solved independently from each other by their individual solver.
-Master algorithms control the data exchange between subsystems and the
-synchronization of all simulation solvers (slaves).
-Simple master algorithms,
-as well as more sophisticated ones are both supported.
+the subsystems are solved independently by internal means, see below.
+Master algorithms control the data exchange and the
+synchronization between subsystems (FMUs).
+
 Note that the master algorithm itself is not part of the FMI standard.
 
+_FMI for Hybrid Co-Simulation_
+
+The Hybrid Co-Simulation interface adds a number of features primarily to allow for more sophisticated master algorithms that aim at more efficient and robust simulations.
+Such additional features are raising events between communication time points using synchronous and asynchronous clocks or sharing values between communication points to allow for improved interpolation of data.
+
```

Comment:
The master algorithm is responsible to advance the overall simulation time and trigger execution of synchronous and asynchronous external events for an FMU.

Reply:
Comment: We need a good description of events and clocks and their similarities and differences in the documentation.